.github/CODEOWNERS

```diff
@@ -1,3 +1,4 @@
 # automatically requests pull request reviews for files matching the given pattern; the last match takes precendence
 
 *       @spacetelescope/stcal-maintainers
+*       @schlafly
```

Comment:
Why not just add him to `stcal-maintainers`?

Reply:
What @kmacdonald-stsci said.

Reply:
Good point, but I can't add new members to the team: https://github.com/orgs/spacetelescope/teams/stcal-maintainers
```suggestion
```

@kmacdonald-stsci could you add @schlafly please?

Reply:
Done.

Reply:
> What @kmacdonald-stsci said.

Is there any other edit we should do with @spacetelescope/stcal-maintainers ?  Should @cshanahan1 still be a maintainer and should anyone else be added as a maintainer, especially since I will be out for a while starting Monday?

Reply:
I'm not sure, but be aware that anyone in the team (or in `CODEOWNERS`) will be pinged by notifications for every PR by default